spec/factories/lessons.rb

```diff
@@ -4,13 +4,19 @@
     sequence(:title) { |n| "test lesson#{n}" }
     sequence(:position) { |n| n }
     github_path { '/lesson_course/lesson_title.md' }
-    content { 'content' }
     identifier_uuid { SecureRandom.uuid }
 
     trait :project do
       is_project { true }
       accepts_submission { true }
       has_live_preview { true }
     end
+
+    after(:create) do |lesson|
```

Comment:
This is a small hack, mostly to avoid adding `content` associations to all the existing specs. Happy to do that though if we're happy with the overall implementation